File: Question6/Question6.js
'use strict';

process.stdin.resume();
process.stdin.setEncoding('utf-8');

let inputString = '';
let currentLine = 0;

process.stdin.on('data', inputStdin => {
    inputString += inputStdin;
});

process.stdin.on('end', _ => {
    inputString = inputString.replace(/\s*$/, '')
        .split('\n')
        .map(str => str.replace(/\s*$/, ''));

    main();
});

function readLine() {
    return inputString[currentLine++];
}

function solution(n, flows) {
    // Write solution here
    return "";
}

function main() {
    var n = parseInt(readLine());

    var flows = readLine().split(' ').map(Number);

    var answer = solution(n, flows);
    console.log(answer);
}

File: Question6/Question6.java
import java.io.*;
import java.util.*;

class Solution {
    static String solution(int n, ArrayList<Integer> flows) {
        // Write solution here
        return "";
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int n = Integer.parseInt(br.readLine());

        String str = br.readLine();
        String[] numsStr = str.split(" ");
        ArrayList<Integer> flows = new ArrayList<Integer>();
        for (String numStr : numsStr) {
            flows.add(Integer.parseInt(numStr));
        }

        String answer = solution(n, flows);
        System.out.println(answer);
    }
}
